### PR TITLE
add tastaturberuf/contao-image-copyright-bundle

### PR DESCRIPTION
**Pull request by Tastaturberuf**
and fix the readme :)

### CODE REVIEW

**Comment by Tastaturberuf** [comment]
Ich habe die de.txt neu sortieren lassen weil ich zu faul bin zu überlegen wo welches Wort hingehört. Generell halte ich das mit dem Linter für großen Käse. mMn.

File: docs/de/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ meta/[vendor]/[paket-name]
     - ru.yml
     - ...
     - logo.svg (optional)
-    - composer.json (nur für private Pakete und optional. [Details](#öffentliche-vs-privateproprietäre-pakete))
+    - composer.json (nur für private Pakete und optional. Siehe Abschnitt "Öffentliche vs. private/proprietäre Pakete")
```
Comment (fritzmg) [inline docs/de/README.md:21]: This should be part of a separate PR.

File: linter/whitelists/fr.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+que
+si
+stock
+toutes
+un
+une
+utile
+utilisez
+vous
+à
```
Comment (fritzmg) [inline linter/whitelists/fr.txt:38]: A lot of these words shouldn't really needed to be whitelisted, should they? 😁

Comment (Tastaturberuf) [reply linter/whitelists/fr.txt:38]: I dont know, i put all words i used in my .yml in this file.

Comment (fritzmg) [reply linter/whitelists/fr.txt:38]: Well, the point of the whitelist is not to put _all_ words in there. The spell checker is there to help spelling errors. If you add all words (including words with spelling errors for example) then it is of not much help ;). Why did you add all words?

Comment (Tastaturberuf) [reply linter/whitelists/fr.txt:38]: Because that is my best french and i dont see an error. i hate this linter shit and have no just to resolve one error, commit,  and another, commit, and another, commit. i have things to do. 

Why is that so important that we spellcheck the descriptions? nobody cares, everyone is self responsible for this. thats my opinon.

Comment (Tastaturberuf) [reply linter/whitelists/fr.txt:38]: One of the first error where "there is noch fr.txt". And then, i added one. done.

Comment (Tastaturberuf) [reply linter/whitelists/fr.txt:38]: when i use in future the translator for nl, ru, pl, how the hack can i say that is correct typo? its good enough for the user and that is wat matters.

**Comment by Tastaturberuf** [comment]
Hat sich mal einer die deutsche Whitelist angeguckt? Da steht auch nen Haufen Käse drin.

**Comment by Tastaturberuf** [comment]
So. Lets go. 🚀

**Comment by Tastaturberuf** [comment]
Ich werde das ganze jetzt mal Testweise in den Translator werfen und ne andere Sprache wählen. Aber dann anderer PR.

**Comment by fritzmg** [comment]
> Hat sich mal einer die deutsche Whitelist angeguckt? Da steht auch nen Haufen Käse drin.

zB?

> So. Lets go. 🚀

Sorry, but now there are lots of unrelated changes in the whitelist 😬 😁

**Comment by Tastaturberuf** [comment]
Willst du mich verarschen?

**Comment by Tastaturberuf** [comment]
Adress
Besonders
Bildslider
HERE
IOTravelCostsBundle
LinkedIn
MATE
MonitoringClientSensorGoogleMaps
MonitoringScanClientWorkerUpdateGooglemapsInstalles
Plenta
Varone
WOW
standardmässig
